_posts/en/newsletters/2020-07-29-newsletter.md

```diff
@@ -93,8 +93,10 @@ release candidates.*
 [Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
 (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
 
-- [Bitcoin Core #19473][] net: Add -networkactive option
-  FIXME:bitschmidty
+- [Bitcoin Core #19473][] adds suport for `networkactive` as both a command line
+  start-up and configuration file option. Toggling this option enables or
+  disables all P2P network activity. `setnetworkactive` has been available as an
+  RPC command and in the GUI since 0.14.0.
```

Comment:
For this final sentence, I suggest rewriting to describe what can be done via the RPC and GUI and then mentioning how, e.g.: "After the node has been started, network activity can be toggled using the existing `setnetworkactive` RPC or the network activity button in the GUI."

Reply:
Thanks for this suggestion